daemon/jobs.go

```diff
@@ -81,13 +83,76 @@ func (r *RPC) StopKillSwitch() error {
 	}
 
 	if cfg.KillSwitch {
-		if err := r.netw.UnsetKillSwitch(); err != nil {
-			return fmt.Errorf("unsetting killswitch: %w", err)
+		// do not unset killswitch rules if system is in shutdown or reboot
+		systemd := internal.IsSystemd()
+		shutdownIsActive := (systemd && r.systemShutdown.Load()) ||
+			(!systemd && internal.IsSystemShutdown())
```

Comment:
Maybe `internal.IsSystemShutdown()` should be also checked o on systemd, just in the case the dbus monitor not catching the shutdown message?

Reply:
Ok, this is maybe a good idea.